debian/changelog

```diff
@@ -1,4 +1,4 @@
-igsc (0.9.3-0ubuntu1~24.10~ppa2) oracular; urgency=medium
```

Comment:
Please increment the ppaX number instead. `dch -i` should handle this bit for you

Reply:
Also, please use a more descriptive commit message